test/OpenSpaceToolkit/Astrodynamics/Flight/Profile.test.cpp

```diff
@@ -1138,77 +1177,27 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Flight_Profile, CustomPointing)
     }
 }
 
-class OpenSpaceToolkit_Astrodynamics_Flight_Profile_Parametrized
+class OpenSpaceToolkit_Astrodynamics_Flight_Profile_Parametrized_SimpleTargetTypes
```

Comment:
simply a refactor of this test (shuffling around code), no change to its contents